### PR TITLE
correctly serialize the retired status of subjects

### PR DESCRIPTION
**Pull request by camallen**
Closes: https://github.com/zooniverse/Panoptes-Front-End/issues/3077

Rename the enum field to not override the instance `blank?` method.
``` ruby 
irb(main):041:0> SubjectWorkflowStatus.retired.by_subject_workflow(subject.id, workflow.id).present?
=> false
irb(main):042:0> SubjectWorkflowStatus.retired.by_subject_workflow(subject.id, workflow.id)         
=> #<SubjectWorkflowStatus id: .....>
irb(main):053:0> SubjectWorkflowStatus.retired.by_subject_workflow(subject.id, workflow.id).blank?  
=> true
```

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

this was returning false even though an AR instance was returned.

### CODE REVIEW

**Comment by marten** [comment]
Interesting, any idea why that is? 

A test case to make sure we don't refactor it back might be a good idea.

**Comment by camallen** [comment]
yeah - just figured it out, it's due to an enum field :blank declared on that model. I'll update my PR to rename the enum field so that it doesn't overwrite the blank? method.

**Comment by marten** [comment]
How awesome that Rails doesn't do a nice warning that you probably don't want to shoot that foot. \_(ツ)_/¯

File: app/operations/workflows/retire_subjects.rb [hunk 1]
```diff
@@ -1,6 +1,9 @@
 module Workflows
   class RetireSubjects < Operation
```
Comment (marten) [inline app/operations/workflows/retire_subjects.rb]: This operation should (for now) rewrite incoming "blank" reasons to "nothing_here". Nero sends that, so we'd break the API if we don't. It's not a public API method as far as I can tell from the docs, so this is just so we decouple the Nero update from this one.

Comment (camallen) [inline app/operations/workflows/retire_subjects.rb]: Got it, will update.

Comment (camallen) [inline app/operations/workflows/retire_subjects.rb]: updated

**Pull request merged**